packages/table/src/common/loadableContent.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+import { Classes } from "@blueprintjs/core";
+
+export interface ILoadingProps {
+    /**
+     * If true, show a fixed height loading skeleton instead of the cell content.
+     * @default false
+     */
+    loading?: boolean;
```

Comment:
if this interface will be shared across several components then the docs should be generalized so they can be reused in different contexts (for instance, don't talk about fixed-height or cells).

or, just define this prop separately in each interface with more complete and relevant documentation.